EntAppProject/src/main/resources/templates/JBFindAJob.html

```diff
@@ -2,39 +2,22 @@
 
 <html>
 
-<head>
-
-<title>Find A Job</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body>
- 
-<h1><span>Job</span>Book</h1>
-
-<h2>Find a Job Near You</h2>
- 
-<form action= "/JBUserProfile">
-
-Location:
-<br>
-<input style=position:left; type="text" id="userBio">
-<br> 
-
-Seeking a Payment of:
-<br>
-<input style=position:left; type="text" id="userSkills">
-
-<br>
-
-<button type="submit">Submit</button>
-
-<!--Planning to have a job board list added to all places where-->
-
-</form>
- 
-</body>
-
-</html>
\ No newline at end of file
+    <head>

```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.